File: Projekti/Python-Native-Benchmark/model/post.py
class Post: 

    def __init__(self, id, user_id, title, content, timestamp):
        self.post_id = id
        self.user_id = user_id
        self.title = title
        self.content = content 
        self.timestamp = timestamp 
        self.tags = []

    def add_tags(self, tags):
        self.tags = tags

    def add_tag(self, tag):
        self.tags.append(tag)
        

File: Projekti/EntityFramework-Benchmark/Model/Post.cs
using System;
using System.Collections.Generic;

namespace EntityFramework_Benchmark.Model;

public partial class Post
{
    public int PostId { get; set; }

    public int? UserId { get; set; }

    public string? Title { get; set; }

    public string? Content { get; set; }

    public DateTime? Timestamp { get; set; }

    public virtual User? User { get; set; }

    public virtual ICollection<Tag> Tags { get; set; } = new List<Tag>();
}
